xformers/ops/fmha/ck.py

```diff
@@ -0,0 +1,514 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from dataclasses import replace
+from enum import Enum
+from functools import partial
+from typing import Any, List, Mapping, Optional, Set, Tuple, Union
+
+import torch
+
+from ..common import get_xformers_operator, register_operator
+from . import attn_bias
+from .attn_bias import (
+    AttentionBias,
+    BlockDiagonalCausalLocalAttentionFromBottomRightMask,
+    BlockDiagonalCausalLocalAttentionMask,
+    BlockDiagonalCausalMask,
+    BlockDiagonalCausalWithOffsetPaddedKeysMask,
+    BlockDiagonalMask,
+    LowerTriangularFromBottomRightLocalAttentionMask,
+    LowerTriangularFromBottomRightMask,
+    LowerTriangularMask,
+    LowerTriangularMaskWithTensorBias,
+)
+from .common import (
+    AttentionBwOpBase,
+    AttentionFwOpBase,
+    Context,
+    Gradients,
+    Inputs,
+    _attn_bias_apply,
+    check_lastdim_alignment_stride1,
+)
+
+
+def _minimum_gemm_alignment(inp: Inputs) -> int:
+    return 1
```

Comment:
For cuda/NV GPU, we have gemm alignment like https://github.com/facebookresearch/xformers/blob/main/xformers/ops/fmha/cutlass.py#L57-L60 . Wonder here why it is set to 1?